src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -94,12 +101,16 @@ def start_monitoring(self):
                     encrypted_message = self._build_encrypted_message(
                         message_dict, loop_start_time)
 
-                    self.mailsender.sendmail(message_string=encrypted_message,
-                                             recipients=self.valid_recipient_emails)
+                    self._send_mail(mime_message=encrypted_message,
+                                    recipients=self.valid_recipient_emails)
                     self.logger.info('Message %s sent successfully.', file_name)
 
                     os.remove(os.path.join(self.outbox_path, file_name))
 
+                if self.netcheck_broadcast.check():
+                    self.logger.info('Broadcast consumed. Flushing sendmail queue.')
```

Comment:
"Broadcast consumed" probably won't mean much to an end user. Can you describe this in terms that will help the end user know what is going on?